treeherder/perf/management/commands/store_subtest_as_foreign_key.py

```diff
@@ -0,0 +1,41 @@
+from optparse import make_option
```

Comment:
store_subtest_as_foreign_key isn't a great name for a command, I think it's likely to confuse people

Could you rename the file to `migrate_perf_signature_subtest_schema.py`?